### PR TITLE
Various catalyst cleaning

### PR DESCRIPTION
**Pull request by mirh**
~~Moved all the driver file names to _tarball var, since (for as much as predictably sounding) newer releases follow no convention.~~
Reminder info is in commits description.

### CODE REVIEW

**Comment by Tids** [comment]
Looks like a nice cleanup/bugfix. There should be no downside on this :+1:

**Comment by mirh** [comment]
Would be lovely if it could be merged before phil the flash starts to put his hands on 4.15

**Comment by philmmanjaro** [comment]
So this is just modifications on the cosmetics on the PKGBUILDs, no rebuilds needed. I can merge it at any time.

**Comment by mirh** [comment]
Mhh, yeah, cosmetics is really the right word I guess.

File: catalyst-utils/PKGBUILD [hunk 1]
```diff
@@ -8,14 +8,14 @@ pkgver=15.201.1151
 pkgrel=11
 epoch=1
 _pkgver=15.201.1151
-_tarball=15.9-linux-installer-15.201.1151
+_tarball=amd-catalyst-15.9-linux-installer-15.201.1151-x86.x86_64
```
Comment (jonathonf) [inline catalyst-utils/PKGBUILD]: Could this use `${_pkgver}` instead of explicitly setting the version (15.201.1151)?

Comment (mirh) [reply catalyst-utils/PKGBUILD]: Nope. As I was saying in the OP, for as much as that name may sound "predictable", it isn't. 
See for example [here](http://support.amd.com/en-us/download/desktop?os=Linux+x86_64). Totally different convention.

Comment (jonathonf) [reply catalyst-utils/PKGBUILD]: What _pkgver would you use for https://www2.ati.com/drivers/linux/radeon-crimson-15.12-15.302-151217a-297685e.zip ?

Comment (jonathonf) [reply catalyst-utils/PKGBUILD]: Actually, looking at the PKGBUILD as a whole, I don't see the need for a separate _tarball variable at all - just put the full download path in the `source` array.

Comment (mirh) [reply catalyst-utils/PKGBUILD]: Mhh.. I had been thinking of that, though I eventually just said that hey, at least source line looks neatly not-overly-long this way 😛

Comment (mirh) [reply catalyst-utils/PKGBUILD]: Done

**Comment by mirh** [comment]
And there we are. 2 weeks in advance for nothing. :/

**Comment by jonathonf** [comment]
Phil said he could merge it, so...

**Comment by mirh** [comment]
Yes, but it misses 4.15 now. 
I'd have rebased if given another day.

**Pull request merged**